calico-cloud_versioned_docs/version-19-2/release-notes/index.mdx

```diff
@@ -5,6 +5,13 @@ title: Release notes
 
 # Calico Cloud release notes
 
+## July 23, 2024 (version 19.4.1)
+
+## Bug fixes
+
+* Adjusted install for new changes on clusters from AKS with managed Calico with Kubernetes 1.29. A new resource was added by AKS that will cause the operator on previous versions to stop
+  managing resources resulting in installs failing.
```

Comment:
I might suggest something along these lines:

Previously, for AKS clusters with managed Calico, the installer failed to connect clusters to Calico Cloud because of changes to the resources deployed by AKS. We made changes to Calico Cloud installer. Now you can successfully install Calico Cloud on AKS cluster with managed Calico.